inspirehep/modules/records/receivers.py

```diff
@@ -371,3 +372,17 @@ def check_if_record_is_going_to_be_deleted(sender, *args, **kwargs):
         else:
             record = get_db_record(pid_type, control_number)
             soft_delete_pidstore_for_record(record.id)
+
+
+def populate_experiment_suggest(sender, json, *args, **kwargs):
+    """Populates experiment_suggest field with the link to the experiment"""
+    accelerator_experiments = json.get("accelerator_experiments")
+    if accelerator_experiments:
+        for accelerator_experiment in accelerator_experiments:
+            experiment = accelerator_experiment.get("experiment")
+            record = accelerator_experiment.get("record")
+            if experiment and record:
+                accelerator_experiment.update({"experiment_suggest": {
+                    "input": experiment,
+                    "payload": {"$ref": record.get("$ref")}
+                }})
```

Comment:
BTW this methods looks perfectly testable: you just assert that the right side effects on `json` have happened. You should be able to find examples in `tests/unit/records/test_record_receivers.py` for this.